docs/source/guides/user/chapters/configuring.rst

```diff
@@ -150,7 +150,7 @@ be parsed into a proper type internally on Avocado.
 
 Currently Avocado supports the following data types for the configuration options:
 ``string``, ``integer``, ``float``, ``bool`` and ``list``. Besides those
-primitives data types Avocado also supports custom data types that can be used
+primitive data types Avocado also supports custom data types that can be used
 by a particular plugin.
```

Comment:
I assume you checked for typos in a quick spell checker too and didn't find anything else. Good!